framework/extensions/src/test/resources/org/apache/cloudstack/framework/extensions/util/testmanifest.yaml

```diff
@@ -0,0 +1,111 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+apiVersion: cloudstack.apache.org/v1
+kind: OrchestratorExtension
+
+metadata:
+  name: test
+  displayName: Test Extension
+  description: >
+    Test extension via the Orchestrator Extension Framework.
+  version: 0.1.0
+  maintainer: "Test Maintainer <maintainer@test.com>"
+  homepage: "https://github.com/maintainer/test"
+
+source:
+  type: git
+  url: "https://github.com/maintainer/test"
+  refs: "branchName"
+
+spec:
+  type: Orchestrator
+
+  compatibility:
+    cloudstack:
+      minVersion: 4.23.0
+
+  entrypoint:
+    language: python
```

Comment:
should this be relevant to the extension framework? is only the executable relevant, not its implementation?